x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/sync_jobs/sync_jobs.tsx

```diff
@@ -63,44 +69,58 @@ export const SyncJobs: React.FC<SyncJobsProps> = ({ connector }) => {
     }
   }, [connectorId, selectedSyncJobCategory]);
 
+  useEffect(() => {
+    if (selectedIndexType === 'content-index') {
+      setSelectedSyncJobCategory('content');
+    } else {
+      setSelectedSyncJobCategory('access_control');
+    }
+  }, [selectedIndexType]);
+
   return (
     <>
       {shouldShowAccessSyncs && (
-        <EuiButtonGroup
-          legend={i18n.translate(
-            'xpack.enterpriseSearch.content.syncJobs.lastSync.tableSelector.legend',
-            { defaultMessage: 'Select sync job type to display.' }
-          )}
-          name={i18n.translate(
-            'xpack.enterpriseSearch.content.syncJobs.lastSync.tableSelector.name',
-            { defaultMessage: 'Sync job type' }
-          )}
-          idSelected={selectedSyncJobCategory}
-          onChange={(optionId) => {
-            if (optionId === 'content' || optionId === 'access_control') {
-              setSelectedSyncJobCategory(optionId);
-            }
-          }}
-          options={[
-            {
-              id: 'content',
-              label: i18n.translate(
-                'xpack.enterpriseSearch.content.syncJobs.lastSync.tableSelector.content.label',
-                { defaultMessage: 'Content syncs' }
-              ),
-              ...(errorOnContentSync ? { iconSide: 'right', iconType: 'warning' } : {}),
-            },
-
-            {
-              id: 'access_control',
-              label: i18n.translate(
-                'xpack.enterpriseSearch.content.syncJobs.lastSync.tableSelector.accessControl.label',
-                { defaultMessage: 'Access control syncs' }
-              ),
-              ...(errorOnAccessSync ? { iconSide: 'right', iconType: 'warning' } : {}),
-            },
-          ]}
-        />
+        <>
+          <AccessControlIndexSelector
+            onChange={setSelectedIndexType}
+            valueOfSelected={selectedIndexType}
+            indexSelectorOptions={[
+              {
+                description: i18n.translate(
+                  'xpack.enterpriseSearch.content.searchIndex.documents.selector.contentIndex.description',
+                  {
+                    defaultMessage: 'Browse content syncs',
+                  }
+                ),
+                error: errorOnContentSync ? true : false,
+                title: i18n.translate(
+                  'xpack.enterpriseSearch.content.searchIndex.documents.selector.contentIndex.title',
+                  {
+                    defaultMessage: 'Content syncs',
+                  }
+                ),
+                value: 'content-index',
+              },
+              {
+                description: i18n.translate(
+                  'xpack.enterpriseSearch.content.searchIndex.documents.selector.accessControl.description',
+                  {
+                    defaultMessage: 'Browse document level security syncs',
+                  }
+                ),
+                error: errorOnAccessSync ? true : false,
```

Comment:
NitPick- non blocking: `errorOnAccessSync` and `errorOContentSync` should be enough. Ternary operator (` something ? return1 : return2;`) is not necessary

Reply:
Makes sense. Already fixed.